packages/paste-core/components/side-panel/stories/index.stories.tsx

```diff
@@ -1,28 +1,36 @@
+import { Anchor } from "@twilio-paste/anchor";
+import { Avatar } from "@twilio-paste/avatar";
```

Comment:
suggestion(non-blocking): probably best to keep these eslint-disables at the top of the file so eslint doesn't get angry about these 2 imports

Reply:
Great suggestion, addressed in latest PR